### PR TITLE
[Lua] Correct an issue preventing final chest from being in Temenos-West

### PR DESCRIPTION
**Pull request by CatsEyeXI**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Correct an issue preventing final chest from being opened.

## Steps to test these changes

Enter Temenos West and fight to the final floor. Open chest and profit.

### CODE REVIEW

**Comment by Xaver-DaRed** [comment]
This doesn't seem completely right, for some reason.

Wouldnt this be fixed by just adding this npc ID `16928830`
inside `ITEM_CRATES` table in script/zones/Temenos/IDs.lua file?

**Comment by jmcmorris** [comment]
So I believe this issue was caused by a regression you can see here: https://github.com/LandSandBoat/server/commit/7cdb557b6a3ec9fe2cb0f5a5d3977a39a533c385#diff-e047edfc1f16bb6b5b73212bba36d4f7c9efbb2504ce4ad551e8cfe52040e582L529-R452https://github.com/LandSandBoat/server/commit/7cdb557b6a3ec9fe2cb0f5a5d3977a39a533c385#diff-e047edfc1f16bb6b5b73212bba36d4f7c9efbb2504ce4ad551e8cfe52040e582L529-R452

Did you try the fix with only the `[ID.TEMENOS_WESTERN_TOWER.npc.LOOT_CRATE] =` change?

File: scripts/battlefields/Temenos/temenos_western_tower.lua [hunk 1]
```diff
@@ -449,7 +458,7 @@ content.loot =
         },
     },
 
-    [7] =
+    [ID.TEMENOS_WESTERN_TOWER.npc.LOOT_CRATE] =
```
Comment (Xaver-DaRed) [inline scripts/battlefields/Temenos/temenos_western_tower.lua:452]: I tested jmcmorris suggestion and it worked.

This change here is the only change needed

Comment (jmcmorris) [reply scripts/battlefields/Temenos/temenos_western_tower.lua:452]: 👍

**Review by Xaver-DaRed — APPROVED** [review]

**Pull request merged**